browser/package.json

```diff
@@ -46,5 +46,10 @@
     "react-window": "^1.7.1",
     "styled-components": "^4.2.0",
     "whatwg-fetch": "^3.5.0"
+  },
+  "devDependencies": {
+    "jest-styled-components": "^6.0.0",
+    "ts-migrate": "^0.1.30",
```

Comment:
Is this a dependency so that we can periodically re-ignore? Should we add an npm script for easily/consistently doing that?

Reply:
The `reignore` command can be helpful, but I also found a lot of manual editing and cleanup to be necessary afterwards, so the payoff of using it is not as high as you might think. I'd suggest that we actually remove `ts-migrate` as a dependency after we merge this PR, and if we think we want it in the future we can always reinstall it on a moment's notice.